File: RemoveDupSortedArray.java
//Time Complexity- O(n)
//Space Complexity- O(1)
//Successfully ran on leetcode

class Solution {
 public int removeDuplicates(int[] nums) {
     if(nums.length<2) return nums.length;
     int count=1;
      int j=1;
     for(int i=1; i< nums.length; i++){
         
        
         if(nums[i]== nums[i-1]){
             count++;
         }
         else count=1;
         
        if(count<=2){
            nums[j]= nums[i];
            j++;
        } 
         
     }
     return j;
 }
}

File: remove_dup_sorted_array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        s=0
        count1=1
        for i in range(0,len(nums)):
            if i!=0 and nums[i-1]==nums[i]:
                count1+=1
                print(str(i)+"in same"+str(count1))
            else:
                count1=1

            if count1<=2:
                print(str(i)+"in s incre")
                nums[s]=nums[i]
                s+=1
            
        return s

        